internal/dataplane/client/client.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kong/blixt/pkg/vars"
+)
+
+// NewDataPlaneClient provides a new client for communicating with the grpc API
+// of the data-plane given a function which can provide the API endpoint.
+func NewDataPlaneClient(ctx context.Context, c client.Client) (BackendsClient, error) {
+	endpoints, err := GetDataPlaneEndpointsFromDefaultPods(ctx, c)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(endpoints) < 1 {
+		return nil, fmt.Errorf("no endpoints could be found for the dataplane API")
+	}
+
+	if len(endpoints) > 1 {
```

Comment:
oooh this will be fun :) Any issues to track this?

Reply:
Yes, https://github.com/Kong/blixt/issues/10 tracks this.